experiments/openebs/openebs-target-network-loss/jiva_controller_network_delay.yaml

```diff
@@ -0,0 +1,70 @@
+--- 
+- name: Identify the jiva controller pod belonging to the PV
+  shell: > 
+    kubectl get pods -l {{ jiva_controller_pod_label }}
+    -n {{ a_ns }} -o jsonpath='{.items[?(@.metadata.labels.openebs\.io/persistent-volume=="{{ pv_name }}")].metadata.name}'
```

Comment:
@shubhamchaudhary  from where you are getting the value for the var {{ pv_name }}

Reply:
from test_prerequisites.yml , We are using pv_name in many places so the idea is to derive pv name once in starting of the experiment.